### PR TITLE
lxqt-admin-time: Make it work on non-systemd GNU/Linux systems

### PR DESCRIPTION
**Pull request by luis-pereira**
Patch made by: Chris Clayton https://github.com/chris2553
Closes https://github.com/lxqt/lxqt-admin/issues/136.

Reference:
https://github.com/lxqt/lxqt-admin/issues/136

### CODE REVIEW

File: lxqt-admin-time/CMakeLists.txt [hunk 1]
```diff
@@ -58,6 +58,11 @@ lxqt_translate_desktop(DESKTOP_FILES
 lxqt_app_translation_loader(lxqt-admin-time_QM_LOADER ${PROJECT_NAME})
 #************************************************
 
+find_package(Systemd)
```
Comment (luis-pereira) [inline lxqt-admin-time/CMakeLists.txt:61]: @chris2553 Where is the Systemd CMake Module ?
LXQt doesn't provide one, CMake also doesn't provide it and the systemd package itself also doesn't provide it.
`find_package()` will always fail and `lxqt-admin-time` will always be built without Systemd support.

**Comment by chris2553** [comment]
@luis-pereira  Good point. Attached is a replacement patch that uses pkgconfig instead of cmake to check for systemd.

[lxqt-admin-0.16.0-nosystemd-2.patch.zip](https://github.com/lxqt/lxqt-admin/files/5533949/lxqt-admin-0.16.0-nosystemd-2.patch.zip)

**Comment by luis-pereira** [comment]
@chris2553 Can you pls test ?
Tested under Void Linux, worked Ok.

**Comment by chris2553** [comment]
Thanks for polishing and updating my patch. It works fine here on my system which is based on the no-systemd variant of
Linux From Scratch.

Tested-by: Chris Clayton <chris2553@googlemail.com>

**Comment by luis-pereira** [comment]
Tested it and it's working Ok. 
@all Can somebody take a look ?

**Comment by luis-pereira** [comment]
Merging to get more feedback.

**Pull request merged**